ports/stm32/eth.h

```diff
@@ -29,12 +29,19 @@
 typedef struct _eth_t eth_t;
 extern eth_t eth_instance;
 
-void eth_init(eth_t *self, int mac_idx, int phy_addr);
+void eth_init(eth_t *self, int mac_idx, int phy_addr, int phy_type);
 void eth_set_trace(eth_t *self, uint32_t value);
 struct netif *eth_netif(eth_t *self);
 int eth_link_status(eth_t *self);
 int eth_start(eth_t *self);
 int eth_stop(eth_t *self);
 void eth_low_power_mode(eth_t *self, bool enable);
 
+enum {
```

Comment:
Please put this enum at the top of this file.

Reply:
Done.